src/editor/codemirror/config.ts

```diff
@@ -74,4 +74,7 @@ export const editorConfig: Extension = [
   EditorState.tabSize.of(indentSize), // But hopefully not used!
   indentUnit.of(" ".repeat(indentSize)),
   python(),
+  lspLinter(),
+  autocompletion(),
+  linter(() => []),
```

Comment:
Do we need this?

Reply:
We need the linter as the language server client sets the diagnostics directly. Maybe we can reconsider but enough for now.